bundles/org.openhab.binding.boschshc/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -8,4 +8,11 @@
 	<description>This is the binding for Bosch Smart Home.</description>
 	<connection>local</connection>
 
+	<discovery-methods>
+		<discovery-method>
+			<service-type>mdns</service-type>
+			<mdns-service-type>_homeconnect._tcp.local.</mdns-service-type>
```

Comment:
Where can I find more info what `_homeconnect` means? Where is this identifier evaluated?

Reply:
Homeconnect devices seem to advertise themselves via mDNS with the _homeconnect application on the _tcp transport an the local domain..

![image](https://github.com/openhab/openhab-addons/assets/893994/3207fc00-dae6-4098-aab5-a6c0037376d9)

Reply:
But Homeconnect devices are handled by the `homeconnect` binding, not the `boschshc` one?

Reply:
You are right. Good catch. I will change it.

Reply:
The Bosch SHC binding implements a generic mDNS lookup with this pattern: `_http._tcp.local.`. It then tries to reach a specific URL on the provided hosts to check whether it is a Bosch Smart Home Controller. I'm not sure if `_http._tcp.local.` would be the correct pattern in this case or if this would basically suggest the binding to all users (which would probably not be intended).

Reply:
> not sure if _http._tcp.local. would be the correct pattern in this case or if this would basically suggest the binding to all users

Indeed. The `_http._tcp.local.` application is very generic and that alone would result in the binding being falsely suggested too often. So we need to apply an additional more exclusive test. The approach used by Bosch SHC of "call home and check credentials" is a model used also by other addons, so I am aware of it. However it is probably not something that we could or should implement in OH core, since it is probably not sufficiently manufacturer agnostic. Ideally we should try to see if there is an mDNS custom attribute within the `_http._tcp.local.` response that is a specific for Bosch SHC. => Perhaps you could run an mDNS browser test on your LAN to see if there are some such attributes analog to the example of  another `_http._tcp.local.` on my own LAN (as shown below).

![image](https://github.com/openhab/openhab-addons/assets/893994/a4b5936a-851c-4a9d-afcf-1a605639343f)

Reply:
I just did the following..
- copied the addon.xml entries from BoschSHC over to HomeConnect => so the suggestion finder would now properly suggest that addon
- did NOT (yet) delete the entries from BoschSHC .. pending feedback from @david-pace => so the suggestion finder would still wrongly suggest that addon; and if @david-pace cannot find a precise search marker in the mDNS then I will finally delete the entries for this addon

Reply:
@andrewfg I found the same app you are using, and searched for `_http._tcp`. I see the Bosch Smart Home Controller but it only shows

* IP Address (it is not the local network address, but a `169.254.*` address
* Port: `8443`
* Service Type: `_http._tcp`

and no attributes. Do I have to configure something to see attributes, or does this mean that the controller has no specific attributes?

Reply:
@david-pace thankyou for checking this. Unfortunately it seems that the info from the mdns discovery does not have any markers to exclusively identify the bosch shc eco system. So I shall have to exclude this one (at least for the time being) from this auto suggestion process.